.github/workflows/docs.yaml

```diff
@@ -0,0 +1,56 @@
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+name: Docs
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+  # Pull requests to main trigger this workflow
+  # However, the deployment is skipped (see check on github.ref)
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Configure
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install ford>=7.0.9 # Notes in included markdown do not work in 7.0.8 
```

Comment:
You might want to create a `requirements.txt` for this.

Reply:
Good point on a `requirements.txt`, I'll add it so that others can easily build as well using the correct version.